model-optimizer/extensions/front/mxnet/batch_dot_ext.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.MatMul import MatMul
+from mo.front.extractor import FrontExtractorOp
+from mo.front.mxnet.extractors.utils import get_mxnet_layer_attrs
+from mo.graph.graph import Node
+
+
+class BatchDotExt(FrontExtractorOp):
```

Comment:
Could you write the description of the operation?

Reply:
Done

Reply:
Could you write more detail description of the operation, please? More precisely, attributes of operations, conditions on input shapes, dimensions that are used to calculate, etc.